scio-smb/src/main/java/org/apache/beam/sdk/extensions/smb/SortedBucketTransform.java

```diff
@@ -222,7 +246,7 @@ public void accept(ValueT t) {
     }
   }
 
-  private static class MergedBucket implements Serializable {
+  public static class MergedBucket implements Serializable {
```

Comment:
```suggestion
static class MergedBucket implements Serializable {
```

Reply:
This needs to be accessible in `SortMergeBucketScioContextSyntax`